src/Admin/Tools/SendMailToUser/SendMailToUserController.php

```diff
@@ -51,4 +53,67 @@ public function sendAction(Request $request): Response
 
     return new Response('OK - message sent');
   }
+
+  public function previewAction(Request $request): Response
+  {
+    $template = (string) $request->query->get('template');
+
+    switch ($template) {
+      case 'confirmation':
+        return $this->renderConfirmation($request);
+      case 'reset':
+        return $this->renderReset($request);
+      case 'basic':
+        return $this->renderBasic($request);
+    }
+
+    return new Response('404');
```

Comment:
The second param is the status code:

return new Response('Not Found', Response::HTTP_NOT_FOUND);